.github/workflows/validate-examples.yml

```diff
@@ -0,0 +1,27 @@
+name: Validate AsyncAPI Examples
+
+on:
+  push:
+    branches:
+      - script-ci-spec-validation # added temporarily for testing purpose
+      # - master
+
+  pull_request_target:
+    types: [opened, reopened, synchronize, edited, ready_for_review]
+
+jobs:
+  validate-examples:
+    name: Validate Spec Examples
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+      - name: Install AsyncAPI CLI
+        run: npm install -g @asyncapi/cli
+      - name: Validate AsyncAPI documents
+        run: |
+          find examples/ \( -path 'examples/social-media/*' -prune \) -o -type f \( -name "*.yml" -o -name "*.yaml" \) -exec asyncapi validate {} \;
```

Comment:
Nice oneliner 👏 👏 

There is only a couple of issues here:

1. The path to skip should be `'examples/social-media/common`.
2. I don't find `prune` option to be very readable. Instead, you can use the `-not` operator. E.g. `-not -path 'examples/social-media/common'`
3. With `-exec` option you can't parallelise the work, so everything will run sequentially. 
Instead, try piping the find output to `xargs`. 

Applying that, will look like:

```sh
find examples -type f \( -name "*.yml" -o -name "*.yaml" \) -not -path 'examples/social-media/common' | xargs -P 10 -L 1 asyncapi validate
```

Reply:
Additionally, please add a comment on top of the command explaining why we are skipping that dir.

Reply:
> Nice oneliner 👏 👏

Thank you! :grin: To be honest, I'm still learning bash so somethings may take time to understand :sweat_smile: 

> There is only a couple of issues here:
> 
>     1. The path to skip should be `'examples/social-media/common`.
> 
>     2. I don't find `prune` option to be very readable. Instead, you can use the `-not` operator. E.g. `-not -path 'examples/social-media/common'`
> 
>     3. With `-exec` option you can't parallelise the work, so everything will run sequentially.
>        Instead, try piping the find output to `xargs` and the `-P` arg.
> 
> 
> Applying that, will look like:
> 
> ```shell
> find examples -type f \( -name "*.yml" -o -name "*.yaml" \) -not -path 'examples/social-media/common' | xargs -P 10 -L 1 asyncapi validate
> ```

Thank you for the suggestions, will try to understand and implement the same. :pray:

Reply:
> Additionally, please add a comment on top of the command explaining why we are skipping that dir.

Sure, I'll do that.